test/passes/Oz_fuzz-exec_all-features.wast

```diff
@@ -1,5 +1,6 @@
 (module
  (type $struct (struct i32))
+ (type $superstruct (struct i32 f64))
```

Comment:
Isn't this a substruct? Actually MVP.md does not say much about the requirements between subtype relationship between static types from which we derive RTTs, other than this:
> Note: `rtt n? $t1` is not a subtype of `rtt n? $t2`, if `$t1` is merely a subtype of `$t2`; such covariant subtyping would be unsound, since RTTs are used in both co- and contravariant roles (e.g., both when constructing and consuming a reference)

What I'm wondering is, if `rtt n? $t1` is a subRTT of `rtt n? $t2`, is it required to be `$t1 <: $t2`?

Reply:
Ah, yes, I may have mixed english with CS terms :smile: I meant "superstruct" as in "superstructure", something even bigger than the original. I'll rename it.

About that quote, I don't fully understand the issues there (why would it be unsound?) but I think the answer to your last question is true, that is required for memory safety.

Reply:
I don't understand that contravariant thing either. I just copy-pasted it from MVP.md...